speech/grpc/pom.xml

```diff
@@ -112,6 +112,11 @@ limitations under the License.
   <!-- // [START dependency] -->
   <dependencies>
     <dependency>
+      <groupId>com.google.cloud</groupId>
```

Comment:
Is this `gcloud-java` dependency necessary? I don't see where you use anything from this package.